packages/website-eslint/types/index.d.ts

```diff
@@ -1,15 +1,16 @@
 import type { TSESLint } from '@typescript-eslint/utils';
 
-import { analyze } from '@typescript-eslint/scope-manager/dist/analyze';
-import { astConverter } from '@typescript-eslint/typescript-estree/dist/ast-converter';
-import { getScriptKind } from '@typescript-eslint/typescript-estree/dist/create-program/getScriptKind';
+import type { analyze } from '@typescript-eslint/scope-manager/dist/analyze';
+import type { astConverter } from '@typescript-eslint/typescript-estree/dist/ast-converter';
 import esquery from 'esquery';
+import type { configs, rules } from '@typescript-eslint/eslint-plugin';
 
-export interface LintUtils {
-  createLinter: () => TSESLint.Linter;
+export interface EslintUtilsModule {
```

Comment:
While we're here, I'm not sold on "utils" as being the part of the name. This is a pretty important chunk - and there are already multiple things called "utils" or similar. Not sure what an alternate name would be though... "bundled"? "packaged"? 🤔 🤷